content/docs/2.14/scalers/mongodb.md

```diff
@@ -70,6 +72,7 @@ To connect to the MongoDB server, you can provide either:
 
 Or provide more detailed connection parameters explicitly (a connection string will be generated for you at runtime):
 
+- `scheme` - The scheme of the MongoDB server, if using MongoDB Atlas, you can set it to `mongodb+srv`. (Default: `mongodb`)
```

Comment:
We have to add also that's optional (https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler)
```suggestion
- `scheme` - The scheme of the MongoDB server, if using MongoDB Atlas, you can set it to `mongodb+srv`. (Default: `mongodb`, Optional)
```